stats/lib/request-helpers.js

```diff
@@ -8,61 +8,60 @@ const getDayAsISOString = (d) => d.toISOString().split('T')[0]
 export const today = () => getDayAsISOString(new Date())
 
 /**
+ * @template {import('./typings.d.ts').DateRangeFilter} FilterType
```

Comment:
I am not sure if this will work exactly as intended. I'll tweak it as needed while enabling type checks.

https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#template